orttraining/orttraining/test/session/training_session_test_utils.h

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#include "gtest/gtest.h"
+#include "orttraining/core/optimizer/gist_encode_decode.h"
+#include "test/providers/provider_test_utils.h"
+#include "test/framework/test_utils.h"
+#include "core/common/path_utils.h"
+#include "core/providers/cpu/cpu_execution_provider.h"
+#include "core/session/environment.h"
+#include "orttraining/core/framework/distributed_run_context.h"
+#include "orttraining/models/runner/training_runner.h"
+#include "orttraining/core/session/training_session.h"
+#include "orttraining/test/optimizer/horizontal_parallel_test_utils.h"
+
+#include "orttraining/training_ops/cpu/controlflow/event_pool.h"  // TODO: move with PipelineBatchPlanner
+
+#ifdef USE_CUDA
+#include "core/providers/cuda/cuda_execution_provider.h"
+#endif
+
+using namespace onnxruntime::logging;
```

Comment:
don't put using directives in header files as it pollutes the namespace for files that include this